examples/lang/calls/rust-toolchain

```diff
@@ -0,0 +1 @@
+nightly-2019-05-21
```

Comment:
We are about to drop `rust-toolchain` files so I think you can also just remove it for this PR.

Reply:
Please delete `rust-toolchain` file.